tracee-ebpf/Makefile

```diff
@@ -119,7 +119,7 @@ $(OUT_DIR)/tracee.bpf.%.o: $(BPF_SRC) $(LIBBPF_HEADERS) | $(OUT_DIR) $(bpf_compi
 	rm $(@:.o=.ll)
 else
 #docker no, CO:RE yes
-$(OUT_DIR)/tracee.bpf.core.%.o: $(BPF_SRC) $(LIBBPF_HEADERS) $(CMD_CLANG)
+$(OUT_DIR)/tracee.bpf.core.o: $(BPF_SRC) $(LIBBPF_HEADERS) $(CMD_CLANG)
```

Comment:
doesn't respect OUT_BPF variable.

Reply:
I didn't get to review the original CORE PR in time so making a comment about it here: I think the user-defined CORE variable wasn't the best choice. core object is a different artifact than the regular object file, so it should be a simple make target. unlike the DOCKER variable which produces the same artifacts in a different way, the CORE variable augments the build process to produce an entirely new artifact. I think this might be unnecessarily complicating the makefile. don't want it to block anything now, just sharing my thoughts